sklearnex/_utils.py

```diff
@@ -63,3 +63,69 @@ def get_patch_message(s):
 def get_sklearnex_version(rule):
     from daal4py.sklearn._utils import daal_check_version
     return daal_check_version(rule)
+
+
+def get_patch_str(name=None, verbose=True):

```

Comment:
I think functional for global patch move to global folder

Reply:
agree, thanks